### ISSUE TITLE
UNable to load Modules

### ISSUE THREAD

**Issue by Gregm3970** [Myopengrid/mwi#32]
I have the web interfaced all set up, and it working fine. But when I tried to load the thumbnail and osregistry modules (to make custom avatars)  I get

Whoops!

Server Error: 500 (Internal Server Error)

What does this mean?

Something went wrong on our servers while we were processing your request. We're really sorry about this, and will work hard to get this resolved as soon as possible.

Perhaps you would like to go to our home page?

I know the web interface is communicating with my data base. I have added new accounts regions and it is working fine.

Any Ideas ?

**Comment by GwynethLlewelyn** [comment]
For those still googling for answers to their issues with Mwi... just give up, nobody is maintaining the code any longer, and it's very hard to believe that anyone ever will, because it relies on such outdated frameworks and programming styles of a far-gone era, that the work to backpatch everything to get it running again is far greater than starting a new project from scratch, which is what I did — after spending _hours_ trying to rewrite the base Laravel framework to comply with contemporary versions of PHP. It's a nightmare; Laravel is too huge for a single person to patch. The version of Laravel used in Mwi was rendered obsolete a decade ago or so — Laravel has evolved through several generations, and the current codebase is totally different, with no possible migration path.

In other words, the work to even track down a _single_ instance of the `500` error is an insane amount of work, most of it totally unrelated to the Mwi application itself, but rather to the underlying Laravel framework.
